optuna/storages/_rdb/storage.py

```diff
@@ -1356,7 +1333,9 @@ def get_current_version(self) -> str:
     def get_head_version(self) -> str:
 
         script = self._create_alembic_script()
-        return script.get_current_head()
+        head = script.get_current_head()
+        assert head is not None
+        return head
```

Comment:
I'm just curious about the motivation for this change. When will this assertion be failed?

Reply:
I don't know the reason either, but without this change it seems to fail the type check.

Reply:
https://github.com/optuna/optuna/runs/6680783237?check_suite_focus=true

Reply:
Good catch! It looks out of the scope of this PR, and himkt-san fixed the problem at #3608. Could you merge master branch and resolves the conflict?

Reply:
Done.